HW3/Task3.py

```diff
@@ -0,0 +1,28 @@
+# В большой текстовой строке подсчитать количество встречаемых слов и вернуть 10 самых частых.
+# Не учитывать знаки препинания и регистр символов.
+# За основу возьмите любую статью из википедии или из документации к языку.
+import string
+
+text = "Переме́нный ток - электрический ток," \
+       " который с течением времени изменяется по величине, обычно и по направлению в электрической цепи." \
+       " Хотя переменный ток часто переводят на английский как alternating current, эти термины не эквивалентны." \
+       "Переменный ток (англ. Аlternating Сurrent) - электрический ток," \
+       " который с течением времени изменяется по величине и направлению (или только по величине или направлению)." \
+       " Основные характеристики переменного тока это: форма сигнала," \
+       " амплитудные и временные характеристики силы тока и напряжения." \
+       "Магнитное поле всегда возникает вокруг движущихся электрических зарядов," \
+       " или при взаимодействии тел, обладающих магнитным моментом." \
+       " Поскольку современные электрические сети используют в основном переменный электрический ток," \
+       " то магнитное поле изменяет своё значение и направление периодически."
+
+result = {}
+text = text.translate(str.maketrans('', '', string.punctuation)).split()
+
+for i in text:
+    if i.lower() in result.keys():
+        result[i.lower()] += 1
+    else:
+        result[i.lower()] = 1
```

Comment:
Во-первых, метод `lower` лучше применить либо 19, либо в 21 строке. Во-вторых, проверка на вхождение и так происходит среди ключей. Наиболее удобен для подсчета количество встречаемых символов метод `get`.
```suggestion
for i in text.lower():
    result[i] = result.get(i, 0) + 1
```